### ISSUE TITLE
pull first and last commit

### ISSUE THREAD

**Issue by xdamman** [OpenCollective/opencollective-jobs#3]
I see that right now in the `group.data` you only record the number of commits per contributor, can you also add the `firstCommitAt` and `lastCommitAt`?
E.g. 

```
{ "githubContributors": { 
  "tj": { 
    "firstCommitAt": "2012-01-01 12:00:11", 
    "lastCommitAt": "2014-01-01 12:00:11",
    "commits": 1292
  },
 ...
}
```

**Comment by boneskull** [comment]
we're limited to getting the last commit b/c of our strategy of pulling contributions

**Comment by boneskull** [comment]
@xdamman Sorry, I made a mistake here; what I said before only applies to external contributions (the "DigitalOcean" use case).  

In this case, we are getting contributions via the Repositories API.  There's no timestamp info; only a list of contributors, and how many contributions.  So to do this, I'd need to either:

a) Combine this data with info from the Events API, which _may or may not contain either one of these dates_ due to its limitations on historical data
b) Implement some portion of #5 

Unfortunately, I'm not going to have time to do _either_ of these things right now.

**Comment by xdamman** [comment]
Get something done with the best data that you have without having to refactor everything.
If you can only have the last commit so be it, that will already be useful data to have.

**Comment by boneskull** [comment]
@xdamman That would be option a), which I still won't have time to do right now.

**Comment by xdamman** [comment]
How long would that take and when would you have time for that?

**Comment by boneskull** [comment]
@xdamman see slack reply

**Comment by xdamman** [comment]
Would it be easy to add `avatar`, `website` and `email` (if any) easily to `group.data.githubContributors` ?

**Comment by boneskull** [comment]
Yes, I believe that would be straightforward, and similar to how I
explained what needs to happen in Slack
